infra/terraform/database.tf

```diff
@@ -34,15 +35,14 @@ resource "azurerm_postgresql_flexible_server_database" "postresql_database" {
   server_id = module.postresql_database[0].database_server_id
 }
 
-
-# Demo purposes only: assign current user as admin to the created DB
+# Demo purposes only: assign current user as admin for the primary DB
```

Comment:
Should you make it so that there is an 'azd env set' before they can use 'azd up' to reflect their awareness? As the user will probably NOT read the terraform?